common/test/acceptance/pages/studio/users.py

```diff
@@ -254,6 +254,7 @@ def click_delete(self):
         self.wait_for_element_visibility('.prompt', 'Prompt is visible')
         self.wait_for_element_visibility('.prompt .action-primary', 'Confirmation button is visible')
         self.q(css='.prompt .action-primary').click()
+        self.wait_for_element_absence('.page-prompt .is-shown', 'Confirmation prompt is hidden')
         wait_for_ajax_or_reload(self.browser)
```

Comment:
Is the ``wait_for_ajax_or_reload`` still necessary since you're now waiting for the confirmation to be hidden? I prefer to have explicit waits for aspects that are expected, rather than vague 'wait until everything seems to have settled down' which more often than not doesn't quite do what we expect.